docs/source-app/glossary/storage/drive_content.rst

```diff
@@ -4,13 +4,24 @@
 About Drives
 ************
 
-Lightning Drive storage makes it easy to share files between LightningWorks so you can run your Lightning App both locally and in the cloud without changing the code.
+Lightning Drive storage makes it easy to share files between LightningWorks (Work) so you can run your Lightning App both locally and in the cloud without changing the code.
```

Comment:
Add cross references to the relevant classes

Reply:
I'll do this once I have some examples.